src/main/java/com/sampleweb/pages/PCVideoGamesPage.java

```diff
@@ -0,0 +1,219 @@
+package com.sampleweb.pages;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
+
+public class PCVideoGamesPage extends HomePage {
+
+  public static String PATH = "https://www.amazon.co.uk/s/ref=nb_sb_noss?url=search-alias%3Dvideogames&field-keywords=";
+
+  ////////////////////////////////////////////////////////////////////////////////////////////
+  ///////////////////////               Web Elements                   ///////////////////////
+  ////////////////////////////////////////////////////////////////////////////////////////////
+  @FindBy(xpath = "//h1[text()='PC and Video Games']")
+  private WebElement title;
+
+  @FindBy(name = "url")
+  private WebElement dropDown;
+
+  @FindBy(xpath = "//h4[text()='Global Store']")
+  private WebElement globalStore;
```

Comment:
Change 'globalStore' to 'globalStoreHeader' to better match the 'avgReviewHeader' variable.